test/assets/import/invalid-package/api/create-service.json

```diff
@@ -0,0 +1,3 @@
+{
+    "serviceName": "{{ .context.service }}"
```

Comment:
We are using `[[` and `]]` for templateing in files to avoid clashing with the templating synatax of other services (like webhook-service)

Reply:
This is an example of an invalid package, so the wrong syntax was added by design.

Reply:
Hmm I would prefer to have just some bogus data in there that's easily distinguished from real data.

Reply:
So you would like the formatting to be correct and the data to be non-existing (wrong mappings, etc.)—any suggestions from data that is easily distinguished from valid data?

Reply:
You could use something like this:
```suggestion
    "serviceName": "{{ .context.service.not-templated }}"
```

Reply:
Just added the changes